packages/ui/src/forum/queries/forum.graphql

```diff
@@ -122,6 +122,26 @@ fragment ForumThreadFields on ForumThread {
     }
   }
   visiblePostsCount
+  posts {
+    id
+    createdAt
+    isVisible
+    text
+    authorId
+    edits {
+      createdAt
+    }
+    status {
+      __typename
+    }
+    threadId
+    thread {
+      categoryId
+    }
+    author {
+      ...MemberFields
+    }
+  }
```

Comment:
I don't think we should go with this solution:
ATM on the Joystream General category it ~6x the size of the thread requests from `137kb` to `851kb` but the 30 posts request saved by this solution were between `4kb` to `10kb` so already the page queries a lot more data to start with. And then it repeats the thread request every 6 seconds. On top of that the thread request go all the way to several megabytes on some pages.

A bigger request can often be better than a lot of smaller one but here I really don't think this is worth it.

Reply:
I have removed "posts" from thread and updated the code. please review and give me feedback